File: StarMines/Data/Python/logic.py
def behave():
    s = 3
    b = Behavior()

    dataT = c_int * 0
    data = dataT()

    for i in range(1,51):
        b.spawn(b"Blue", float(random.randint(0, 500)),
                 float(random.randint(0, 500)), data, 0)

    for i in range(1,11):
        b.spawn(b"Green", float(random.randint(0, 500)),
                   float(random.randint(0, 500)), data, 0)

    b.spawn(b"Red", float(random.randint(0, 500)),
             float(random.randint(0, 500)), data, 0)

    b.deactivate()

behave()

File: StarMines/TestData/Pandemic/Lua/Logic.lua
function Logic(bc)
	for i=1,30 do
		local x = math.random(0,770)
		local y = math.random(0, 570)
		bc:spawn("Green", x, y, "", "default")
	end

	for i=1,40 do
		local x = math.random(0,770)
		local y = math.random(0, 570)
		bc:spawn("Blue", x, y, "", "default")
	end

	local x = math.random(0,770)
	local y = math.random(0, 570)
	bc:spawn("Red", x, y, "", "default")

	bc:deactivate()
end